pkg/controller/zerocapacity/zero_controller.go

```diff
@@ -210,14 +210,14 @@ func (z *Controller) initializeResources(request reconcile.Request, instance Res
 	if err := z.Client.Get(ctx, clusterKey, infinispan); err != nil {
 		z.Log.Info(fmt.Sprintf("Unable to load Infinispan Cluster '%s': %w", clusterName, err))
 		if errors.IsNotFound(err) {
-			return reconcile.Result{RequeueAfter: consts.DefaultWaitOnCluster}, nil
+			return reconcile.Result{Requeue: true, RequeueAfter: consts.DefaultWaitOnCluster}, nil
 		}
 		return reconcile.Result{}, err
 	}
 
 	if err := infinispan.EnsureClusterStability(); err != nil {
 		z.Log.Info(fmt.Sprintf("Infinispan '%s' not ready: %s", clusterName, err.Error()))
-		return reconcile.Result{RequeueAfter: consts.DefaultWaitOnCluster}, nil
+		return reconcile.Result{Requeue: true, RequeueAfter: consts.DefaultWaitOnCluster}, nil
```

Comment:
`Requeue: true` is redundant, from the docs:

```
	// RequeueAfter if greater than 0, tells the Controller to requeue the reconcile key after the Duration.
	// Implies that Requeue is true, there is no need to set Requeue to true at the same time as RequeueAfter.
```

Reply:
Oh, yes, will fix. Thanks, @ryanemerson